src/components/pages/my-space.js

```diff
@@ -70,7 +73,10 @@ function MySpace({ PageTemplate }) {
 }
 
 MySpace.propTypes = {
-  PageTemplate: PropTypes.func.isRequired
+  PageTemplate: PropTypes.func.isRequired,
+  initialState: PropTypes.shape({
+    rooms: PropTypes.arrayOf(roomShape)
```

Comment:
Should this not also be required (I mean the room array)?

Reply:
True, even when we bypass fetching the rooms, the array is still set. Changed, thanks.